recipes/tree-format/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "tree-format" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a538523aa78ae7a4b10003b04f3e1b37708e0e089d99c9d3b9e1c71384c9a7f9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - tree_format
+
+about:
+  home: https://github.com/jml/tree-format
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Python library to generate nicely formatted trees, like the UNIX `tree` command.'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    Python library to generate nicely formatted trees, like the UNIX `tree`
+    command.
+
```

Comment:
Please remove the description below this line

Reply:
Done